### PR TITLE
[FSPE-6621] Added Popover open  on hover example

### PR DESCRIPTION
**Pull request by janibolkvadze**
[FSPE-6621] Added Popover open  on hover example

[FSPE-6621]: https://ctl.atlassian.net/browse/FSPE-6621

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1619/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://2398e0210c43702de8af3934bb34e97f770b6110-chi-pr.enterprise.ctl.io" target="_blank"> https://2398e0210c43702de8af3934bb34e97f770b6110-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1620/ for more details.

**Review by mattnickles — CHANGES_REQUESTED** [review]
@janibolkvadze One minor issue - the Popover is not showing when using key commands. Behavior should match the drop down hover example.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://9263ffd8cd1cff6d8107f68b36994ae59aa624e2-chi-pr.enterprise.ctl.io" target="_blank"> https://9263ffd8cd1cff6d8107f68b36994ae59aa624e2-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1622/ for more details.

File: src/website/views/components/popover/_examples.pug [hunk 1]
```diff
@@ -220,6 +220,72 @@ p.-text
       <button id='popover-1' class="chi-button" data-popover-content='<header class="chi-popover__header"><h2 class="chi-popover__title">Popover title</h2></header><div class="chi-popover__content"><p class="chi-popover__text">Popover content</p></div>'>Popover</button>
       <script>chi.popover(document.getElementById('popover-1'));</script>
 
+h3 Open on hover
```
Comment (jllr) [inline src/website/views/components/popover/_examples.pug:223]: @janibolkvadze examples are not working on IE11, maybe it's related to JS arrow function.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://6c855198fbde59549ea1a79cad911cc18229b960-chi-pr.enterprise.ctl.io" target="_blank"> https://6c855198fbde59549ea1a79cad911cc18229b960-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1627/ for more details.

**Review by jllr — APPROVED** [review]

**Review by mattnickles — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-07ixm8, https://ops.elasticbox.com/services/instances/i-7dkelv, https://ops.elasticbox.com/services/instances/i-79ki0o

**Pull request merged**